assets/js/components/legacy-notifications/util.js

```diff
@@ -111,8 +117,8 @@ export async function getModulesNotifications() {
 		const promise = new Promise( async ( resolve ) => {
 			const { identifier } = module;
 
-			const notifications = removeDismissed(
-				await data.get( TYPE_MODULES, identifier, 'notifications', {}, false ),
+			const notifications = await removeDismissed(
+				await API.get( 'modules', identifier, 'notifications', {}, false ),
```

Comment:
As mentioned above, `nocache: false` is the same as the default of `useCache: true` which can be removed.
```suggestion
				await API.get( 'modules', identifier, 'notifications' ),
```